### PR TITLE
Write some documentation related to our workflow

### LINKED ISSUE #1: Specify a workflow in a dedicated document
Our collaborative workflow will be inspired by the [Rapid Cycle Workflow](https://gitlab.com/softeng-heigvd/teaching-heigvd-pdg-2020/guidelines/-/blob/master/WORKFLOW.md) presented in class. More specifically:

- we will **create an issue** before working on a specific topic;
- we will **estimate the time needed** for resolving the issue;
- we will work on the issue and **create a [Draft Pull Request](https://github.blog/2019-02-14-introducing-draft-pull-requests/)** when we have our first contribution to the issue;
- we will **log the time we spent** resolving or closing the issue with a dedicated tool; and
- we will **merge or close the contribution** after an appropriate **review process**.

We need a document that clearly specifies these steps and can serve as a reference to all team members. This might also include some specific issue templates, that simplify readability and glancing at the content of any contribution. On top of that, the structure of meetings and the sprint details will also be defined there.

**Estimated time :** 1.5 hours (tracked on the issue until we switch to a different tool)
**Actual time (until now):** 1 hour

### PR DESCRIPTION
**Pull request by alexandrepiveteau**
When merged, this pull request will solve #1.

### CODE REVIEW

**Comment by alexandrepiveteau** [comment]
Before merging anything, I would still like to write some documentation related sprints, epics and how we integrate GitHub projects with them.

File: wiki/WORKFLOW.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+- we're using GitHub, in order to centralize all of our services (issues, project management, milestones, hosting, continuous integration);
+- we want to make sysetmatic pull request reviews; and
+- we'll be using an external time tracking system.
+
+### Rapid Cycle Workflow
+
+1. Based on what was agreed on during meetings, file an issue for a task you'd like to accomplish. When choosing a task, choose something that you think should take you **less than 4 hours** to achieve. It's better to aim for a task that's too small rather than a task that's too big !
+	+ Use the dedicated issue template to file that task.
+	+ Fill in our time tracking system the expected duration of this task.
+2. Use your personal fork, and create a branch dedicated to the issue. Once you have your first commit ready, immediatly create a [draft pull request](https://github.blog/2019-02-14-introducing-draft-pull-requests/).
```
Comment (glsubri) [inline wiki/WORKFLOW.md]: I'd say something like "Create a DPR as soon as possible". I do not want to limit that to the first commit as one can be offline and still work (for example).

File: wiki/WORKFLOW.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+3. Whenever you commit your code, log the amount of time that you have spent on your task. Try to be accurate; it will help us estimate future tasks more precisely !
+	+ Follow [this article](https://chris.beams.io/posts/git-commit/) on how to write a good commit messages. The Git history of your contribution will also be looked at during the review.
+	+ Add a description to your commit message if relevant.
+	+ When doing some peer coding, [create a commit with multiple authors](https://docs.github.com/en/enterprise/2.13/user/articles/creating-a-commit-with-multiple-authors).
+	+ If you have a question on your work, write it in the feed of the draft pull request.
+4. Once you are with your contribution, mark your draft as ready. Make sure that merging the pull request will close the issue created in `1.` !
```
Comment (glsubri) [inline wiki/WORKFLOW.md]: "Once you are with" should be "Once you are done with"

File: wiki/WORKFLOW.md (hunk 1 above)
Comment (alexandrepiveteau) [reply wiki/WORKFLOW.md]: Please let me know if 68a5127b6adadc2c651f2837d2008bb3a03daad5 sounds better ^^

File: wiki/WORKFLOW.md (hunk 2 above)
Comment (alexandrepiveteau) [reply wiki/WORKFLOW.md]: Fixed in dc7289b9a40f3b37524a4e3217179d3e684d804c.

**Review by GTZL1 — COMMENTED** [review]
It's all right for me

**Comment by alexandrepiveteau** [comment]
We'll still need to add some a (generated) table of contents.

**Review by GTZL1 — APPROVED** [review]

File: wiki/WORKFLOW.md (hunk 1 above)
Comment (glsubri) [reply wiki/WORKFLOW.md]: Perfect

**Comment by glsubri** [comment]
> We'll still need to add some a (generated) table of contents.

I'll do it in a further PR

**Review by glsubri — APPROVED** [review]

**Review by matt989253 — APPROVED** [review]

**Pull request merged**